src/agent/debugger/src/debugger.rs

```diff
@@ -205,15 +201,15 @@ pub trait DebugEventHandler {
         // Continue normal exception handling processing
         DBG_EXCEPTION_NOT_HANDLED
     }
-    fn on_create_thread(&mut self, _debugger: &mut Debugger, _info: &CREATE_THREAD_DEBUG_INFO) {}
-    fn on_create_process(&mut self, _debugger: &mut Debugger, _info: &CREATE_PROCESS_DEBUG_INFO) {}
-    fn on_exit_thread(&mut self, _debugger: &mut Debugger, _info: &EXIT_THREAD_DEBUG_INFO) {}
-    fn on_exit_process(&mut self, _debugger: &mut Debugger, _info: &EXIT_PROCESS_DEBUG_INFO) {}
-    fn on_load_dll(&mut self, _debugger: &mut Debugger, _info: &LOAD_DLL_DEBUG_INFO) {}
-    fn on_unload_dll(&mut self, _debugger: &mut Debugger, _info: &UNLOAD_DLL_DEBUG_INFO) {}
+    fn on_create_process(&mut self, _debugger: &mut Debugger, _name: &Path, _base_address: u64 ) {}
+    fn on_create_thread(&mut self, _debugger: &mut Debugger) {}
```

Comment:
Isn't there any thread info we'd maybe want to pass here? An ID? Or even just the `hThread`?

Reply:
Use case: I can imagine using this as a site to initialize per-thread symbolic context.

Reply:
I decided to defer that decision until the first use case came up.

Reply:
Works for me.